software/stringtie/merge/meta.yml

```diff
@@ -0,0 +1,28 @@
+name: stringtie_merge
+description: Merges the annotation gtf file and the stringtie output gtf files
+keywords:
+    - merge
+    - gtf
+    - reference
+tools:
+    - stringtie2:
+        description: |
+            Transcript assembly and quantification for RNA-Seq
+        homepage: https://ccb.jhu.edu/software/stringtie/index.shtml
+        documentation: https://ccb.jhu.edu/software/stringtie/index.shtml?t=manual
+input:
+    - stringtie_gtf:
+        type: file
+        description: |
+            Stringtie transcript gtf output(s).
+    - annotation_gtf:
+        type: file
+        description: |
+            Annotation gtf file.
+output:
+    - merged_gtf:
+        type: map
+        description: |
+            Merged gtf from annotation and stringtie output gtfs.
```

Comment:
I know it's obvious but we usually have a `pattern` field here as well, basically describing the file ending

Reply:
I have added the patterns